.mci.yml

```diff
@@ -224,14 +230,21 @@ functions:
     params:
       updates:
         - key: "mongo_url"
-          value: ${mongo_url_prefix}${mongo_url_enterprise_keyword|"enterprise-"}${mongo_url_platform|}2.6.5.${mongo_url_extension|tgz}
+          value: ${mongo_url_prefix}${mongo_url_enterprise_keyword|enterprise}-${mongo_url_platform|}2.6.5.${mongo_url_extension|tgz}
```

Comment:
Some of the buildvariants don't have an enterprise keyword, I'd expect this to construct invalid urls with "--" in them.

I'd agree that it's a bit backwards for the default to be "enterprise-" and override with "" but that's the way it's currently set up for some reason.